src/nvim/CMakeLists.txt

```diff
@@ -508,6 +538,7 @@ function(get_test_target prefix sfile relative_path_var target_var)
 endfunction()
 
 set(NO_SINGLE_CHECK_HEADERS
+  api/ui_events.in.h
```

Comment:
On line 552 the logic skips files ending in `.c.h`. So if `ui_events.in.h` were named `ui_events.c.h` (which would make sense IMO) it would be automatically skipped. Else the logic could also check for `in.h`.

Either of those is preferable since `NO_SINGLE_CHECK_HEADERS` will be eliminated soon.